Jellyfin.Api/Extensions/RequestExtensions.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using MediaBrowser.Model.Entities;
+using MediaBrowser.Model.Querying;
+using Microsoft.AspNetCore.Http;
+
+namespace Jellyfin.Api.Extensions
+{
+    /// <summary>
+    /// Request Extensions.
+    /// </summary>
+    public static class RequestExtensions
```

Comment:
None of these are extensions

Reply:
What do you propose the file / path be named?